### PR TITLE
Place secret hash into collector.conf

### PR DESCRIPTION
**Pull request by rzvoncek-ds**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -97,8 +100,8 @@ ifdef COLLECTOR_SECRETSMANAGER_SECRET
 		echo "I will now add the key to the Secrets Manager" ; \
 		AWS_ACCESS_KEY_ID=${COLLECTOR_SECRETSMANAGER_KEY} AWS_SECRET_ACCESS_KEY=${COLLECTOR_SECRETSMANAGER_SECRET} aws --region=us-west-2 secretsmanager create-secret --name ${KEY_FILE_NAME} --description "Reuben collector key" --secret-string file://${KEY_FILE_NAME} ;\
 	else \
-		echo "Secret exists in Secrets Manager with ${SECRET_EXISTS}" ; \
-		echo "Issue $(subst /,-,$(ISSUE)) already already exists. Not creating a new secret." ; \
+		echo "Secret exists in Secrets Manager with ${SECRET_EXISTS} so will fetch it." ; \
+		AWS_ACCESS_KEY_ID=${COLLECTOR_SECRETSMANAGER_KEY} AWS_SECRET_ACCESS_KEY=${COLLECTOR_SECRETSMANAGER_SECRET} aws --region=us-west-2 secretsmanager get-secret-value --secret-id ${KEY_FILE_NAME} | jq .SecretString | xargs printf > ${KEY_FILE_NAME} ;\
```
Comment (michaelsembwever) [inline Makefile:104]: `jq` is a new dependency, so it needs the line 93
```
@(command -v openssl >/dev/null 2>&1) || { echo >&2 "openssl needs to be installed"; exit 1; }
```
replaced with
```
@(command -v openssl >/dev/null 2>&1) || { echo >&2 "openssl needs to be installed"; exit 1; }
@(command -v jq >/dev/null 2>&1) || { echo >&2 "jq needs to be installed"; exit 1; }
```

**Review by mieslep — APPROVED** [review]

**Comment by michaelsembwever** [comment]
k8s test broke …?

**Comment by mieslep** [comment]
k8s tests fixed on master.

**Comment by rzvoncek-ds** [comment]
Rebased on master. Should bring in the CI build fixes.

**Pull request merged**